requirements-base1.txt

```diff
@@ -1,4 +1,4 @@
-notebook==4.0.5
-ipykernel==4.0.3
-Cython==0.23.1
-git+git://github.com/numpy/numpy.git@cee68d1e764711be671003e8d6e57c8049cee5ad#egg=numpy
+notebook
+ipykernel
+Cython
+numpy
```

Comment:
I think it might be good to keep the version numbers, but bump them up to the latest if you've tested them and they seem to work. I want to keep reproducible builds, not sure how to do that easily if we don't keep track of versions here.